### DISCUSSION TITLE
Secondary Protocol v5 doesn't match CPCd Protocol v4

### DISCUSSION THREAD

**Discussion by Mermarky** [b2un0/silabs-multipan-docker#8] [Q&A] [answered]
getting following error when spinning docker:
*** ASSERT *** : FATAL in function 'protocol_version_check' in file /usr/src/cpc-daemon/server_core/server_core.c at line #729 : Secondary Protocol v5 doesn't match CPCd Protocol v4

**Comment by b2un0** [comment, marked as answer]
https://github.com/b2un0/silabs-multipan-docker/wiki/Secondary-Protocol-v5-doesn't-match-CPCd-Protocol-v4
